test/train-sets/ref/cb_similar_aml_spinoff_save.stderr

```diff
@@ -13,15 +13,15 @@ Input label = CB
 Output pred = ACTION_PROBS
 average  since         example        example        current        current  current
 loss     last          counter         weight          label        predict features
-0.333333 0.333333            1            1.0          0:1:1         0:0.33     8436
-0.175000 0.016667            2            2.0          1:1:1         2:0.97    18319
-0.333333 0.491667            4            4.0          1:1:1         0:0.97    23278
-0.293750 0.254167            8            8.0          0:1:1         2:0.97    13862
-0.273958 0.254167           16           16.0          0:1:1         2:0.97     2249
+-0.33333 -0.33333            1            1.0         0:-1:1         0:0.33     8436
```

Comment:
why would loss change on the first example if the CS is not used for loss?

Reply:
I had to change the rewards because the new estimator can only use rewards between 0 and 1, implying a costs between -1 and 0